base/sparse/sparsematrix.jl

```diff
@@ -93,11 +91,15 @@ function Base.showarray(io::IO, S::SparseMatrixCSC;
     pad = ndigits(max(S.m,S.n))
     k = 0
```

Comment:
what's a little funny is this `k` is redundant with the `k` in the for loop below